libcloud/compute/drivers/ec2.py

```diff
@@ -189,6 +189,41 @@
         'disk': 160000,
         'bandwidth': None
     },
+    'm4.large': {
+        'id': 'm4.large',
+        'name': 'Large Instance',
+        'ram': 8192,
+        'disk': 0,
```

Comment:
The instances are EBS only: http://www.ec2instances.info/?selected=m4.large,m4.xlarge,m4.2xlarge,m4.4xlarge,m4.10xlarge